internal/process/process.go

```diff
@@ -45,6 +48,13 @@ func ActivationPID(cfg Configurable) int32 {
 			return -1
 		}
 
+		name, err := pproc.Ppid()
+		if err != nil {
+			multilog.Log(logging.ErrorNoStacktrace, rollbar.Error)(procInfoErrMsgFmt, err)
```

Comment:
Don't want to log to rollbar for something we only looked up for debugging purposes.